app/src/main/java/com/example/sopt_seminar/ui/ProfileFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.sopt_seminar.ui
+
+import android.os.Bundle
+import android.view.View
+import com.example.sopt_seminar.R
+import com.example.sopt_seminar.databinding.ProfileFragmentBinding
+import com.example.sopt_seminar.util.BaseFragment
+
+class ProfileFragment : BaseFragment<ProfileFragmentBinding>(R.layout.profile_fragment) {
+    private val followerFragment = FollowerListFragment()
+    private val repoFragment = RepoFragment()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        childFragmentManager.beginTransaction().replace(R.id.home_fragment_cv, followerFragment)
+            .commit()
```

Comment:
fragment-ktx 이용해서 아래처럼 보다 프래그먼트 클래스만 참조해서 보다 간결하게 표현이 가능합니다..!
```suggestion
childFragmentManager.commit {
    replace<FollowerFragment>(R.id.home_fragment_cv)
}
```

Reply:
이렇게 하는게 프래그먼트 클래스 객체를 직접 만들어서 활용하는 것에 비해 메모리 세이프하기도 하구요..!

Reply:
[94ce90f](https://github.com/SAMSUNG-ZZANG/Galaxy-Tab-Wonyong/pull/4/commits/94ce90fb48c5e946e740388ba9e490eb2f613c06) 수정했습니다!!